openapi-module/README.md

```diff
@@ -0,0 +1,210 @@
+    model: "gpt-3.5-turbo",
+    maxTokens: 1500,
+    temperature: 0.7,
+    promptTemplate: function(endpoints) {
+        return `
+        You are given the following API endpoints from an OpenAPI document:
+
+        ${JSON.stringify(endpoints, null, 2)}
+
+        Please generate a Flex scenario JSON file that includes:
+        ...
+        `;
+    },
+    outputDir: '../../temp',
+    outputFileName: 'flex-scenario-gpt.flex',
+};
+```
+
+## Usage
+
+### Generating `.flex` Files and cURL Commands
+
+You can generate `.flex` files and cURL commands using the scripts provided.
+
+#### Static Approach (Faker)
+
+1. Run the script with `--useGPT=false` to use Faker for generating data.
+2. Customize field mappings in `field-mapping.js` if needed.
+3. Run the scripts.
+
+#### AI Approach (OpenAI GPT)
+
+1. Run the script with `--useGPT=true` to use GPT for generating data.
+2. Ensure your API key is set and the `promptTemplate` is configured in `config.js`.
+3. Run the scripts.
+
+### Running the Scripts
+
+### Generate cURL Commands
+
+Generate cURL commands based on your OpenAPI file:
+
+```sh
+npm run generate-curl -- --openApiFilePath=sample/sample-openapi.yaml --outputFilePath=./temp/curl-commands.sh
+```
+
+### Generate Flex Scenarios
+
+Generate Flex scenarios:
+
+```sh
+npm run generate-flex -- --openApiFilePath=sample/sample-openapi.yaml --outputFilePath=./temp/flex-scenario.flex --useGPT=true --gptOutputFilename=my-custom-scenario.flex
+```
+
+You can omit the `--useGPT=true` and `--gptOutputFilename` arguments to use default settings, which will generate the file as `flex-scenario.flex`.
+
+### Generate Both cURL Commands and Flex Scenarios
+
+Generate both cURL commands and Flex scenarios:
+
+```sh
+npm run generate-all -- --openApiFilePath=sample/sample-openapi.yaml --curlOutputFilePath=./temp/curl-commands.sh --flexOutputFilePath=./temp/flex-scenario.flex --useGPT=true --gptOutputFilename=my-custom-scenario.flex
+```
+
+### Customizing Script Execution
+
+You can control the generation process via command-line arguments:
+
+```sh
+npm run generate-flex -- --openApiFilePath=sample/sample-openapi.yaml --outputFilePath=./temp/flex-scenario.flex --useGPT=true --gptOutputFilename=my-custom-scenario.flex
+```
+
+### Params Explained:
+
+- **--openApiFilePath**: Path to your OpenAPI YAML file. Required for all generation scripts.
+  - Example: `--openApiFilePath=sample/sample-openapi.yaml`
+
+- **--outputFilePath**: Path to save the generated .flex file or cURL commands. Required for generating .flex files or cURL commands.
+  - Example: `--outputFilePath=./temp/flex-scenario.flex`
+
+- **--curlOutputFilePath**: Path to save the generated cURL commands. Required when generating cURL commands, especially with generate-all.js.
+  - Example: `--curlOutputFilePath=./temp/curl-commands.sh`
+
+- **--flexOutputFilePath**: Path to save the generated .flex scenarios. Required when generating Flex scenarios, particularly with generate-all.js.
```

Comment:
Is this flex-scenario.flex able to be uploaded in the desktop-app? It's still not clear to me which file I should use in the desktop-app.